core/indexer/process/transactions/serialize.go

```diff
@@ -0,0 +1,229 @@
+package transactions
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+
+	"github.com/ElrondNetwork/elrond-go/core/indexer/types"
+)
+
+// SerializeScResults -
+func (tdp *txDatabaseProcessor) SerializeScResults(scResults []*types.ScResult, bulkSizeThreshold int) ([]bytes.Buffer, error) {
+	var err error
+	var buff bytes.Buffer
+
+	buffSlice := make([]bytes.Buffer, 0)
+	for _, sc := range scResults {
+		meta := []byte(fmt.Sprintf(`{ "index" : { "_id" : "%s" } }%s`, sc.Hash, "\n"))
+		serializedData, errPrepareSc := json.Marshal(sc)
+		if errPrepareSc != nil {
+			log.Warn("indexer: marshal",
+				"error", "could not serialize sc results, will skip indexing",
+				"hash", sc.Hash)
+			continue
+		}
+
+		// append a newline for each element
```

Comment:
the code is swarming with exactly the same useless comment.

Reply:
refactored.